build/pgo-client/Dockerfile

```diff
@@ -1,9 +1,12 @@
 ARG BASEOS
 ARG BASEVER
 ARG PREFIX
+FROM ${PREFIX}/pgo-base:${BASEOS}-${BASEVER}
+
 ARG PGVERSION
 ARG BACKREST_VERSION
-FROM ${PREFIX}/pgo-base:${BASEOS}-${BASEVER}
+ARG PACKAGER
+ARG DFSET
```

Comment:
Do we need these in here even though they are not consumed?

Reply:
Since it is part of the catch-all makefile target and the --build-args is being set for these I wanted to make sure they were being captured in the Dockerfiles.  I believe that if you specify the build-args flag and don't have a corresponding ARG that buildah will spit out a warning.  If you want me to remove them I can, but for consistency sake and to specify every build-arg that was passed into the Dockerfile I would like to keep it